application/src/main/java/org/thingsboard/server/service/edge/rpc/processor/TelemetryEdgeProcessor.java

```diff
@@ -96,6 +94,9 @@ public List<ListenableFuture<Void>> processTelemetryFromEdge(TenantId tenantId,
             if (entityData.hasPostTelemetryMsg()) {
                 result.add(processPostTelemetry(tenantId, customerId, entityId, entityData.getPostTelemetryMsg(), metaData));
             }
+            if (EntityType.DEVICE.equals(entityId.getEntityType())) {
+                deviceStateService.onDeviceActivity(tenantId, new DeviceId(entityId.getId()), System.currentTimeMillis());
```

Comment:
send this via kafka to the correct node. But add a new message to trigger this code in TbCoreConsumer